manticore/ethereum/manticore.py

```diff
@@ -615,11 +615,12 @@ def solidity_create_contract(
                     )
                 else:
                     contract_account = self.create_contract(
-                        owner=owner, init=md._init_bytecode, balance=balance
+                        owner=owner, init=md._init_bytecode, balance=0, gas=gas
                     )
-
                 if contract_account is None:
+                    return None
                     raise EthereumError("Failed to build contract %s" % contract_name_i)
```

Comment:
Isn't this `raise` unreachable?  Is that intentional?

Reply:
Unreachable/unintentional. Was in the middle of deciding what a failed contract creation should do.